test/integration/host_test.rb

```diff
@@ -267,6 +267,9 @@ def multiple_actions_div
   end
 
   describe "hosts index multiple actions" do
+    after do
+      DatabaseCleaner.clean
```

Comment:
Yeah, added this with the comment as the test was failingץ
Read that you have to forcibly clean the DB after tests have ended